### PR TITLE
feat: introduce mock structure for suggested filters

### PR DESCRIPTION
**Pull request by MounirDhahri**
Resolves [ONYX-515]

This PR brings a mock structure for the upcoming suggested filters work. We are starting here by returning a mock then we will introduce the logic in the upcoming tickets.

Screenshot 2023-11-14 at 09 52 49

[ONYX-515]: https://artsyproduct.atlassian.net/browse/ONYX-515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/previewSavedSearch.ts [hunk 1]
```diff
@@ -93,6 +93,14 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
         "URL for a user to view the artwork grid with applied filters matching saved search criteria attributes",
       resolve: resolveHref,
     },
+    suggestedFilters: {
+      description:
+        "Suggested filters for the user to use based on their search criteria",
+      type: new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel)),
```
Comment (MounirDhahri) [inline src/schema/v2/previewSavedSearch.ts:99]: note: I am not sure here if we should allow nulls - I will get to this later when we implement the logic and have a better understanding of what we are going to be returning.

```suggestion
      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel))),
```

Comment (dblandin) [reply src/schema/v2/previewSavedSearch.ts:99]: I think we should always be able to return an empty array or non-null suggestions here.

**Review by dblandin — DISMISSED** [review]
LGTM! - I would make this field non-nullable though 👍

File: src/schema/v2/previewSavedSearch.ts (hunk 1 above)
Comment (MounirDhahri) [reply src/schema/v2/previewSavedSearch.ts]: Cool - I will do it now then

**Pull request merged**